### PR TITLE
Server setup and react installed

### PR DESCRIPTION
**Pull request by Vectormike**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Vectormike/jollof.git"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (mikeattara) [inline package.json]:
```suggestion
  "license": "MIT",
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "bugs": {
+    "url": "https://github.com/Vectormike/jollof/issues"
+  },
+  "homepage": "https://github.com/Vectormike/jollof#readme",
+  "dependencies": {
+    "body-parser": "^1.19.0",
```
Comment (mikeattara) [inline package.json]:
```suggestion
```

Comment (mikeattara) [reply package.json]: body-parser is integrated into express

Comment (Vectormike) [reply package.json]: Yeah, totally forgot

**Pull request merged**